spec/services/workflow_find_service_spec.rb

```diff
@@ -1,6 +1,6 @@
 RSpec.describe WorkflowFindService do
   let(:workflow) { create(:workflow, :with_tenant, :group_refs => [990]) }
-  let(:a_tag) { {:object_type => 'inventory', :app_name => 'topology', :tag_name => '/approval/workflows/abc'} }
+  let(:a_tag) { {:object_type => 'inventory', :app_name => 'topology', :object_id => 'abc'} }
```

Comment:
@hsong-rh This is not a tag its object meta data, something that identifies a remote object